bindings/uniffi/src/web5.udl

```diff
@@ -209,7 +221,6 @@ enum CredentialError {
 };
 
 interface VerifiableCredential {
-  constructor(sequence<string> context, string id, sequence<string> type, string issuer, i64 issuance_date, i64? expiration_date, CredentialSubject credential_subject);
```

Comment:
I think not-using `constructor` everywhere will probably become a rule we follow, and instead rely on the global functions (in this case `new_verifiable_credential`)